controllers/storagecluster/cephcluster.go

```diff
@@ -116,9 +117,17 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	platform, err := r.platform.GetPlatform(r.Client)
+	if err != nil {
+		r.Log.Error(err, "Failed to get platform")
+	} else if platform == v1.IBMCloudPlatformType || platform == IBMCloudCosPlatformType {
+		r.Log.Info("Increasing Mon failover timeout to 900s for %s", platform)
+		cephCluster.Spec.HealthCheck.DaemonHealth.Monitor.Timeout = "900s"
```

Comment:
For readability you could set `15m`, but the result will be the same.
```suggestion
		cephCluster.Spec.HealthCheck.DaemonHealth.Monitor.Timeout = "15m"
```

Reply:
done